packages/module/src/module.ts

```diff
@@ -29,15 +30,36 @@ export default defineNuxtModule<ModuleOptions>({
   async setup({ prefix, componentDir }, nuxt) {
     const COMPONENT_DIR_PATH = componentDir!
     const ROOT_DIR_PATH = nuxt.options.rootDir
+    const UTILS_ALIAS = '@/lib/utils' // Use the same path from the cli for backward compatibility
```

Comment:
This won't break if the user changed `aliases.utils` in `components.json` but it would be rather redundant. We can refactor this when `components.json` options is moved to module options